### ISSUE TITLE
Script generates same experiment ID every time due to fixed random.seed 

### ISSUE THREAD

**Issue by rusiaaman** [replicate/replicate#417]
Minimal script for replication. version is v0.2.2
Run this twice. Experiment created only one.
```
import replicate
import random

random.seed(5)

with open("x.txt", 'w') as f:
    f.write("xyz")
    
experiment = replicate.init(".", params={'x': 'y'})
experiment.checkpoint("x.txt", metrics={"y": 0})
```
---
`hash.py` shouldn't use global state for `random` library. Machine learning experiments need fixed seed for replication.

**Comment by andreasjansson** [comment]
Thanks for this @rusiaaman, great catch! I'll create a fix that includes timestamp in the ID generation function.

**Comment by andreasjansson** [comment]
The fix is released in version 0.2.4
